lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_topic.rb

```diff
@@ -0,0 +1,39 @@
+module GovukPublishingComponents
+  module Presenters
+    class ContentBreadcrumbsBasedOnTopic
+      def self.call(content_item)
+        new(content_item).breadcrumbs
+      end
+
+      attr_reader :content_item
+
+      def initialize(content_item)
+        @content_item = content_item
+      end
+
+      def breadcrumbs
+        breadcrumbs = [{ title: "Home", url: "/" }]
+        return breadcrumbs unless topics.present?
+
+        breadcrumbs << topic_breadcrumb
+        breadcrumbs.reverse!
```

Comment:
I don't think we want to reverse these ones - I've just checked out `/guidance/bond-insurance-policy` locally using this branch with government-frontend and it looks like `Export Finance > Home` when it should be the other way round.

I think we reverse them in other scenarios where we construct them backwards

Reply:
Actually, [that doc I chose](https://www.gov.uk/guidance/bond-insurance-policy) is interesting because it's got topics as it's breadcrumbs, but constructs the hierarchy for the nav by using linked parents. 🤔